src/js/integrations/default.ts

```diff
@@ -104,8 +104,9 @@ export function getDefaultIntegrations(options: ReactNativeClientOptions): Integ
   if (hasTracingEnabled && options.enableAppStartTracking) {
     integrations.push(appStartIntegration());
   }
-  if (hasTracingEnabled && options.enableNativeFramesTracking) {
-    integrations.push(nativeFramesIntegration());
+  const nativeFrames = createNativeFramesIntegrations(hasTracingEnabled && options.enableNativeFramesTracking);
+  if (nativeFrames) {
+    integrations.push(nativeFrames);
```

Comment:
I will suggest changing the name of nativeFrames to nativeFramesIntegration to keep it consistent with the names being pushed below
```suggestion
  const nativeFramesIntegratin = createNativeFramesIntegrations(hasTracingEnabled && options.enableNativeFramesTracking);
  if (nativeFrames) {
    integrations.push(nativeFramesIntegration);
```